src/core/LanguageMenu/LanguageMenu.md

```diff
@@ -0,0 +1,21 @@
+```js
+import {
+  LanguageMenu,
+  LanguageMenuItem
+} from 'suomifi-ui-components';
+
+<LanguageMenu
+  buttonText="In English (EN)"
```

Comment:
I would change this default to be Suomeksi (FI), since that selection only logs the value to console. When clicking it, the language menu seems a bit buggy since "nothing happens".

Reply:
This point is not really applicable anymore since all the items only accept programmatic navigation through onSelect() now